### DISCUSSION TITLE
Some wired repetition happens in transcription.

### DISCUSSION THREAD

**Discussion by catalwaysright** [openai/whisper#192] [Q&A] [answered]
I am testing the medium.en model and I found this wired thing happens in my transcription.
![image](https://user-images.githubusercontent.com/60195620/193094784-03327db7-8b06-4305-910d-b0bef4c91a6c.png)
There should be only one  'I am her mom'. Has someone met the same problem?

**Comment by jongwook** [comment, marked as answer]
This is a failure mode of the sequence-to-sequence architecture (briefly mentioned in the [model card](https://github.com/openai/whisper/blob/main/model-card.md#performance-and-limitations)). Lowering the value of `--compression_ratio_threshold` (2.4 by default) might help.

**Reply by rolaguna** [reply]
Thanks! just want to add: in my case I was also setting `temperature=0`, so instead I let it as default values, decreased the `compression_ratio_threshold` as you suggested, and the problem seems fixed now

**Reply by bradyj04** [reply]
What did you end up lowering the threshold to? Im having the same issue

**Comment by fleek** [comment]
try changing "condition_on_previous_text" to false in transcribe and also lower the "no_speech_threshold"

**Reply by wiseman** [reply]
I would expect that setting `condition_on_previous_text` to `False` would cause an overall significantly decreased accuracy. Am I wrong?

**Reply by fleek** [reply]
I think it will affect if you are transcribing long narrated passages, especially if the sentence crosses the 30 second window.  But my use is for conversations which are basically very short sentences, so I don't expect it to be a accuracy hit.  Anyway, I ran the same file through whisper with the same settings 10 times and it never gave me the exact same output twice, so I cannot gauge whether there is any accurarcy hit or not.

**Reply by nlgtuankiet** [reply]
Hi @jongwook do you have any comments regarding the accuracy of the conversation use case? (< 30 seconds sentence)

**Reply by RichardQin1** [reply]
My audio usually lasts for 30 min. Is this suggestion also applicable. What are the recommended values for setting？thank

**Comment by bjollans** [comment]
An alternative solution that worked for me is processing the file in chunks. I used `pydub` to split the file into 2 minute chunks then used the `large` model with default settings.
